application/ui/src/app/pages/matching/matching.page.html

```diff
@@ -30,6 +30,42 @@
       </form>
     </ion-row>
     <ion-row>
+       <ion-col>
+        <!-- Searchbar with cancel button always shown -->
+        <!--ion-searchbar showCancelButton="always"></ion-searchbar-->
+        <ion-list>
+          <ion-item-divider sticky>
+            <ion-label>
+              Users list
+            </ion-label>
+          </ion-item-divider>
+          <ion-item>
+            <ion-label>Select All</ion-label>
+            <ion-checkbox slot="start" [(ngModel)]="masterCheck" [indeterminate]="isIndeterminate" (click)="checkMaster($event)"></ion-checkbox>
+          </ion-item>
+        </ion-list>
+        <ion-list #scheduleList>
+          <ion-item-group>
+            <ion-item-sliding *ngFor="let user of usersgroups" #slidingItem>
+              <ion-item>
+                <ion-avatar slot="start">
+                  <div [innerHTML]="user.avatar"></div>
+                </ion-avatar>
+                <ion-label>
+                  <h3>{{user.name}}</h3>
+                  <p>
+                    {{user.departement}}
+                  </p>
+                </ion-label>
+                <ion-chip *ngFor="let group of user.groups; let i=index" [color]="ColorsTags[3%(i+1)]" button (click)="tagclick($event)">
+                  <ion-label>{{group}}</ion-label>
+                </ion-chip>
+                <ion-checkbox [(ngModel)]="user.isChecked" (click)="checkEvent($event,user)" color="primary"></ion-checkbox>
```

Comment:
this is called here but below in matching.page.ts you remove the function. can you confirm what is the expected behavior?